modules/auxiliary/server/socks5.rb

```diff
@@ -0,0 +1,68 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'thread'
+require 'rex/proto/proxy/socks5'
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Auxiliary::Report
+
+  def initialize
+    super(
+      'Name'        => 'Socks4a Proxy Server',
```

Comment:
This should say Socks5. I wonder if the version could just be a parameter for a single module instead?